airflow/providers/weaviate/hooks/weaviate.py

```diff
@@ -17,15 +17,20 @@
 
 from __future__ import annotations
 
+import json
 import warnings
 from functools import cached_property
-from typing import TYPE_CHECKING, Sequence
+from typing import TYPE_CHECKING, Literal, Sequence, cast
```

Comment:
When you import TYPE_CHECKING, other imports from typing should be inside its `if` statement

Reply:
Moved the imports with the `if TYPE_CHECKING` code block. `cast` is needed as we are using it outside.